lib/onc_certification_g10_test_kit/bulk_export_validation_tester.rb

```diff
@@ -74,6 +74,21 @@ def stream_ndjson(endpoint, headers, process_chunk_line, process_response)
 
       stream(process_body, endpoint, headers: headers)
 
+      max_redirect = 5
+
+      redirect_url = (response[:headers].find { |header| header.name.downcase == 'location' })&.value
```

Comment:
```suggestion
      redirect_url = request.response_header('location')&.value
```

Reply:
This needs to happen in the `while` loop somewhere to handle multiple redirects, right?